services/f-droid/f-droid.service.js

```diff
@@ -55,64 +64,30 @@ module.exports = class FDroid extends BaseYamlService {
     }
   }
 
-  async fetchYaml(url, options) {
-    const yaml = await this._requestYaml({
+  async fetch({ appId }) {
+    const url = `https://gitlab.com/kitsunyan/fdroidapi/raw/master/applications/${appId}`
```

Comment:
You mentioned in the PR description that this endpoint will be changing once this endpoint gets moved under F-Droid as an official API. What will happen with this endpoint once that transition occurs? Will it redirect to the new endpoint, or will it continue to work for some period after the transition, or will it be a hard cut over? 

Just want to make sure that this doesn't break on us in a couple weeks when that transition occurs

Reply:
Since I'm the one who maintain this API prototype, I'll maintain it until shields.io will be migrated to the official API.

Reply:
That's fantastic. We'll try to keep an eye on the upstream incorporation of the API, but please feel free to drop us a note (either on a GitHub thread or in Discord) when that happens